courses/user/16.md

```diff
@@ -8,4 +8,13 @@ headerDepth: 3
 ## 16. Next Steps
 
 ### 16.1 Take the Class Survey
-Take our 9 question SAF User Class [survey](https://forms.office.com/g/UxNr3nhtcm) to give feedback to fuel class improvement.
\ No newline at end of file
+Take our 9 question SAF User Class [survey](https://forms.office.com/g/UxNr3nhtcm) to give feedback to fuel class improvement.
+
+### 16.2 Take the InSpec Beginner Class
+This class is one of a set of security automation content offered by the MITRE SAF(c) team. If you found this content interesting and you want to learn more about writing InSpec profiles of your own, we encourage you to check out the Beginner Class (shown in the table of contents on the left). You'll be writing your own automated validation tests in no time!
+
+### 16.3 Check Out the Rest of MITRE SAF(c)'s Content
```

Comment:
You could also link them to the class "resources" page which has links to most everything